charts/self-host/values.yaml

```diff
@@ -12,9 +12,9 @@ general:
     className: "nginx"
      ## - Annotations to add to the Ingress resource.
     annotations:
-      nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
-      nginx.ingress.kubernetes.io/use-regex: "true"
-      nginx.ingress.kubernetes.io/rewrite-target: /$1
+      # nginx.ingress.kubernetes.io/force-ssl-redirect: "true"
```

Comment:
These need to be commented out so that they do not merge in if someone is using something like App Gateway instead of NGINX.